middleware/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "sf-fx-middleware",
-  "version": "0.0.1",
+  "version": "0.0.2",
```

Comment:
even though we're not using this version right now.. but I wonder if we should keep consistent with the biuldpack.toml version... or remove it?

Reply:
Yeah, I wasn't sure if this was used - makes sense to keep these in sync.

Reply:
Fixed.